### DISCUSSION TITLE
2022.08.27 브리핑

### DISCUSSION THREAD

**Discussion by swjiae** [itmakesmesoft/Steady-Study#50] [Self-Assessment]
이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.
ex)
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점
이유 : (생략)

**Comment by swjiae** [comment]
이번 주 목표 : 과목평가 70점 이상, 월ˑ수 노션 정리해서 커밋하기, DFS 문제 다시 풀어보기, 운동 2회

점수 : 70점
이유
과목평가 70점 이상 O (최민호 교수님..맹신...)
노션정리 (2/2) 커밋 (0/2)
DFS 문제 다시 풀어보기 X -> IM 대비 우선이라 다음주 장고 때 알고리즘 복습 예정
운동 (2/2)

**Comment by sooyeonlee127** [comment]
이번 주 목표: 주 5회 기술블로그 올리기, 운동 2회

점수: 70점
이유: 
기술블로그 4회 (4/5)
운동 3회 (3/2)

기술블로그 한 회 못했고, 게시글 질도 안좋아서 다음 주에는 블로그에 더 정성을 쏟을 예정
큐 문제 아예 못풀어서 다음주는 강의 내용 이해하고 큐 문제 모두 푸는 것이 목표

**Comment by moonthree** [comment]
이번 주 목표 : WEB APIs, 브라우저 DOM까지 정리해서 올리기

점수 : 0점
이유
알고리즘에서 빠져나오지 못하여 아무것도 하지 못함
다음 주엔 장고니까 열심히 하겠습니다.

**Comment by YoungSeok222222** [comment]
이번주 목표 : SWEA 남은 문제 풀기 + 노션에 올리기 +IM 시험 대비하기

점수: 30 점
이유: 노션에는 업로드 했지만 swea 남은 문제를 못 풀었기 때문에

IM시험 대비 빡세게 준비

**Comment by HyeseungNA** [comment]
이번 주 목표: 수업 복습, 부족한 개념 복습하기, 민코딩 최소 1레벨 풀기
점수 : 60점
이유: 민코딩 매일 풀고 부족한 개념 복습하는 것은 이룸, 하지만 수업 복습은 할 시간이 없었습니다..!

**Comment by giokim12** [comment]
이번주 목표: 민코딩 week1 다 채우기, 수면 3시간 이상
점수: 50점
이유: IM 테스트에 재귀랑 dfs 안나온다길래 그 부분은 안하고 여태 배운거 리뷰함
잠은... ㅎ 쉽지않다...

**Comment by itmakesmesoft** [comment]
이번 주 목표 : 과목평가, 월말 평가 대비 공부. 시험범위 리뷰하고 부족한 부분 다시 풀어보기. 그동안 푼 민코딩 문제 파일로 저장해두기.

점수 : 70점
이유 : 과목평가는 시험범위 리뷰한 덕분에 목표치는 이룸
다만 민코딩 과거 문제를 전부 파일로 옮기지 못했음.
